tests/unit/Test_Schedule.php

```
@@ -0,0 +1,30 @@
+use WP_UnitTestCase;
+
+class Test_Schedule extends WP_UnitTestCase
+{
+    private const REPORT_HOOK = 'wp_statistics_report_hook';
+
+    public function tearDown(): void
+    {
+        wp_clear_scheduled_hook(self::REPORT_HOOK);
+        Option::update('time_report', '0');
+
+        parent::tearDown();
```

PHPStan (2.2.7)

[error] WP_Statistics\Tests\Test_Schedule::tearDown() calls parent::tearDown() but WP_Statistics\Tests\Test_Schedule does not extend any class.


(class.noParent)